.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,91 @@
+name: Renovate
```

Comment:
FYI, we already have renovate repo: https://github.com/homarr-labs/renovate-runner
Also, this workflow doesn't seem have much to do with Renovate, no?
Maybe rename it?

Reply:
i just rename it to Update Chart Metadata